recipes-openamp/rpmsg-examples/rpmsg-utils_1.0.bb

```diff
@@ -0,0 +1,21 @@
+SUMMARY = "RPMsg utilities: utilities for /dev/rpmsg*"
+
+include rpmsg-example.inc
+
+S = "${WORKDIR}/git/examples/linux/rpmsg-utils"
+
+RRECOMMENDS:${PN} = "kernel-module-rpmsg-char"
```

Comment:
Should we add rpmsg-ctrl driver in this list?

Reply:
Ok I will add it

Reply:
This is now done.  I have built with it and verified that the new recommendation is in the package.
Note: now that we have ctrl, char is redundant as ctrl recommends char but I have left them both.

Reply:
@tnmysh @bentheredonethat please take a look again.  I would like to get this done today if possible.